trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0...10]
+  end
```

Comment:
- Let this method check  the length of name. The name should have a maximum of 10 characters. If it's longer it should trim the word.